src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.1.0-beta.5
```

Comment:
@xiang17 Do you think we should no be doing an RC, and we do not anticipate any further changes to this package?

Reply:
I would consider rc with stable OTel.

Reply:
why not do stable of this package, when OTel 1.4 is stable as well ?

Reply:
I am not against stable with stable OTel.
I do not like idea where RC version can be compiled only with exact package version, not even the range.

Reply:
it should be range, given OTel .NET said its the last rc, so we should leave upper bound open.

Reply:
I am not complaining, but the same case was for rc.3.

If possible please merge #994 then this PR.

Reply:
I think doing RC now is a good idea. We should release the RC before we release the stable anyways.

I agree with Piotr. The exact package version is annoying, especially it causes trouble when downstream wants to test against version upgrades.